README.rst

```diff
@@ -10,7 +10,7 @@ Flyteadmin uses a relational style Metadata Store abstracted by `GORM <http://go
 Before Check-In
 ~~~~~~~~~~~~~~~
 
-Flyte Admin has a few useful make targets for linting and testing. Please use these before checking in to help suss out
```

Comment:
mind reverting this? i think it makes more sense as the original :)

Reply:
Haha, I thought that it was a typo. TIL